src/main/java/at/ac/tuwien/kr/alpha/config/IOptionHandler.java

```diff
@@ -0,0 +1,11 @@
+package at.ac.tuwien.kr.alpha.config;
+
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.ParseException;
+
+@FunctionalInterface
+public interface IOptionHandler<T> {
```

Comment:
We don't prefix interfaces with a capital I. Just `grep` through the repo. Just for the sake of consistency I advise against it. Once there's an interface with prefix in the codebase, the second might follow, and then we end up mixing... Sorry if you prefer to have the prefix...

Reply:
Personally rather prefer the variant with prefix - But on the other hand it's really no trouble removing it, so I guess I'm just gonna do that

Reply:
Yeah, I guessed so. Naming is hard. It's just for the sake of consistency really, not to bother you.